.github/workflows/check.yaml

```diff
@@ -1,4 +1,4 @@
-name: Check workflow running linter, unit and functional tests
+name: Tests
```

Comment:
Original name was too long to read which variant was running.

Reply:
"Tests" seems to arbitrary, maybe "Code quality check" or "Quality check"

Reply:
But this is more than quality, this is lints, unit tests, and functional tests.  So tests covers everything.  Also note that with the full path to the job as shown in the github status, it's clear what's running.  The original title is too long and contains redundent information, especially when seen with the job names.

Reply:
(non-native speaker and non-blocking) but those are should be broadly speaking related to "the quality of the code"

Reply:
Good point. :+1:   Ok if we have more discussion later after we finish centrally managing them via the terraform repo?  Then we can pick a good name and update it on all the repos to be consistent.